tests/data/baby_aging.csv

```diff
@@ -0,0 +1,2510 @@
+﻿#Project: asd,,,,

```

Comment:
@pchelle @Yuri05 @PavelBal @abdullahhamadeh 
ok I am havign issues reading this file bacaue it looks corrupted! WTF
That being said, how do you deal with # in csv .using the `comment.char = '#'` does nto seem to do the trick